docs/readthedocs/source/doc/Nano/Howto/Training/TensorFlow/accelerate_tensorflow_training_multi_instance.nblink

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
BigDL-Nano provides ``bigdl.nano.tf.keras.Model`` and ``bigdl.nano.tf.keras.Sequential`` which extend ```tf.keras.Model``` and ```tf.keras.Sequential``` respectively with various optimizations.  To use multi-instance training on a server with multiple CPU cores or sockets, you just replace `tf.keras.Model/Sequential` in your code with `bigdl.nano.tf.keras.Model/Sequential`, and call ``fit`` with specified num_processes.

Reply:
Do we have any tips about setting num_processes? If we have we can add them here, as a Related Reading or a Note.

Reply:
Have added a guide about how to choose number of processes for multi-instance training. @TheaperDeng Please have a look, thank you! https://yuwentestdocs.readthedocs.io/en/nano-howto-training-tf/doc/Nano/Howto/Training/General/choose_num_processes_training.html

Reply:
the guide looks good to me